aspnetcore/security/blazor/server/additional-scenarios.md

```diff
@@ -136,3 +136,62 @@ public class WeatherForecastService
     }
 }
 ```
+
+## Use Open ID Connect (OIDC) v2.0 endpoints
+
+The authentication library and Blazor templates use Open ID Connect (OIDC) v1.0 endpoints. To use a v2.0 endpoint, configure the <xref:Microsoft.AspNetCore.Builder.OpenIdConnectOptions.Authority?displayProperty=nameWithType> option in the `OpenIdConnectOptions`:
+
+```csharp
+services.Configure<OpenIdConnectOptions>(AzureADDefaults.OpenIdScheme, 
+    options =>
+    {
+        options.Authority += "/v2.0";
+    }
+```
+
+Alternatively, the setting can be made in the app settings (*appsettings.json*) file:
+
+```json
+{
+  "AzureAd": {
+    "Authority": "https://login.microsoftonline.com/common/oauth2/v2.0/",
+    ...
+  }
+}
+```
+
+If tacking on a segment to the authority isn't appropriate for the app's OIDC provider, such as with non-AAD providers, set the `Authority` property directly. Either set the property in `OpenIdConnectOptions` or in the app settings file with the `Authority` key.
+
+### Code changes
+
+* The list of claims in the ID token changes for v2.0 endpoints. For more information, see [Why update to Microsoft identity platform (v2.0)?](/azure/active-directory/azuread-dev/azure-ad-endpoint-comparison).
+* The <xref:Microsoft.AspNetCore.Builder.OpenIdConnectOptions.Resource?displayProperty=nameWithType> property isn't required when requesting an access token. Remove the the `Resource` property setting in `OpenIdConnectOptions`:
```

Comment:
There is a difference between `isn't required` vs `it'll break it if it stays there`. I'm not certain which is the right thing here.

If it's really simply not required, but can stay, we should be clear about that.
Otherwise, if it must not be there, as it may break something ,we have to be very clear about this too.

Reply:
It shouldn't stay there. It's a fundamental change in the way that the resource is specified between v1 and v2. The resource is part of the scope in v2. Therefore, I'll make this into an imperative statement. I'll also cross-link it directly to the Azure docs remarks ...

https://docs.microsoft.com/en-us/azure/active-directory/azuread-dev/azure-ad-endpoint-comparison#scopes-not-resources